src/app/Http/Controllers/QuizController.php

```diff
@@ -2,46 +2,32 @@
 
 namespace App\Http\Controllers;
 
+use App\BigQuestion;
+use App\Question;
+use App\Choice;
 use Illuminate\Http\Request;
-use Illuminate\Support\Facades\DB;
 
 
 class QuizController extends Controller
 {
 
     // 問題リスト表示用
-    public function index_links(Request $request) {
+    public function list(Request $request) 
```

Comment:
```suggestion
    public function list() 
```
利用していないのであれば不要な変数は外した方が良いです